### ISSUE TITLE
Icons was prevented by CROS

### ISSUE THREAD

**Issue by amit-gshe** [ovity/octotree#675]
### Description
The icon can not be loaded and prevented by CROS.

### Environment (if bug)

* Octotree version: 3.0.4
* Browser & version: Firefox 66.0.3
* OS & version: Archlinux
* Screenshot: ![图片](https://user-images.githubusercontent.com/7383028/56264091-f4294c00-6117-11e9-9c8f-bc4a3871d7f9.png)
* Console error log:
```
Content Security Policy: 页面设置阻止读取位于 chrome-extension://0f3da9f6-9cba-4585-930a-74ada2d7b287/fonts/fontawesome.woff2 的一项资源("font-src")。
Content Security Policy: 页面设置阻止读取位于 chrome-extension://0f3da9f6-9cba-4585-930a-74ada2d7b287/fonts/octicons.woff2 的一项资源("font-src")。
Content Security Policy: 页面设置阻止读取位于 chrome-extension://0f3da9f6-9cba-4585-930a-74ada2d7b287/fonts/file-icons.woff2 的一项资源("font-src")。
Content Security Policy: 页面设置阻止读取位于 chrome-extension://0f3da9f6-9cba-4585-930a-74ada2d7b287/fonts/devopicons.woff2 的一项资源("font-src")。
Content Security Policy: 页面设置阻止读取位于 chrome-extension://0f3da9f6-9cba-4585-930a-74ada2d7b287/fonts/mfixx.woff2 的一项资源("font-src")。
已拦截跨源请求：同源策略禁止读取位于 https://api.github.com/_private/browser/stats 的远程资源。（原因：CORS 请求未能成功）。[详细了解]
```

**Comment by cmoud94** [comment]
Same here.

- Octotree: 3.0.4
- Firefox: 67.0b11
- OS: Debian buster

Console log:
```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://ab1b5b6c-96d3-4a2a-92c4-ca926b3976d0/fonts/fontawesome.woff2 (“font-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://ab1b5b6c-96d3-4a2a-92c4-ca926b3976d0/fonts/octicons.woff2 (“font-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://ab1b5b6c-96d3-4a2a-92c4-ca926b3976d0/fonts/file-icons.woff2 (“font-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://ab1b5b6c-96d3-4a2a-92c4-ca926b3976d0/fonts/devopicons.woff2 (“font-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://ab1b5b6c-96d3-4a2a-92c4-ca926b3976d0/fonts/mfixx.woff2 (“font-src”).
```

**Comment by Aymeric31** [comment]
Same here with mozilla 66.0.3
And Same console log from @cmoud94

**Comment by 0x6773** [comment]
It's working in chrome but not in Firefox.
Using Firefox 67.0b11 and Chrome 73.0.3683.103.

**Comment by cmoud94** [comment]
> It's working in chrome but not in Firefox.
> Using Firefox 67.0b11 and Chrome 73.0.3683.103 on macos 10.14.4

I can confirm this. I've just tried it out in Chromium 73.0.3683.75 on Debian buster and it's working.

**Comment by nphi** [comment]
Thanks for all your help. I will try to fix this issue ASAP.

**Comment by jimeh** [comment]
I can confirm it's only affecting Octotree 3.0.4. 3.0.3 still shows icons correctly. 

The issue seems to be that the URL used in the icon styles start with `chrome-extension://` instead of `moz-extension://`. So I assume it's a simple fix :)

**Comment by buunguyen** [comment]
It was fixed in 3.0.5. Thanks @jimeh for finding the root cause!

**Comment by 0x6773** [comment]
Thanks, it's fixed now.
To update the extension on firefox manually, go to `about:addons` > `settings` > `check for updates`.
